Installed Node_Module, Pods and Storybook - Testing

Jaydankai opened the pull request:
if you have "warnOnce" error, best way is to start from the beginning - restart your Mac will kill all the current running tasks

sshenis commented:
After making your page, the first thing you need to do is import that comp in STORIES.JS and MAIN.JS. Please follow the instructions in stories.js or on your notes.

Jaydankai commented:
Yes, I added my comp to the index.js of stories folder. I also added my comp in the Main.js and require "Stories". Everything is working properly now!
